### PR TITLE
JFB-2: Adds function to return tickets by Stadium Sector to the TicketService

### PR DESCRIPTION
**Pull request by MadinaBaizhanova**
…tService class.

### CODE REVIEW

**Comment by NikaaSvanidze** [comment]
well done

File: .idea/misc.xml
Comment (deolexx) [inline .idea/misc.xml:1]: You should not store any files from idea folder in git

File: src/TicketService.java [hunk 1]
```diff
@@ -12,6 +13,13 @@ public void addTicket(Ticket ticket) {
         tickets.add(ticket);
     }
 
+    public List<Ticket> getTicketsByStadiumSector(char stadiumSector) {
+        return tickets.stream()
+                .filter(ticket -> ticket.getStadiumSector() == stadiumSector)
+                .collect(Collectors.toList());
```
Comment (deolexx) [inline src/TicketService.java:19]: .collect(Collectors.toList()); could be replaced with just toList()

File: src/TicketService.java [hunk 2]
```diff
@@ -33,5 +41,11 @@ public static void main(String[] args) {
             Ticket ticket = tickets.get(i);
             System.out.println(ticket.toString());
         }
+
+        // Getting tickets by Stadium Sector 'C'
+        List<Ticket> sectorCTickets = ticketService.getTicketsByStadiumSector('C');
+        for (Ticket ticket : sectorCTickets) {
+            System.out.println(ticket.toString());
```
Comment (deolexx) [inline src/TicketService.java:48]: As for now it is acceptable but Using System.out.println  for logging is generally not recommended

**Review by deolexx — APPROVED** [review]

File: .idea/misc.xml
Comment (MadinaBaizhanova) [reply .idea/misc.xml:1]: Thank you for your review! Well noted!

File: src/TicketService.java (hunk 1 above)
Comment (MadinaBaizhanova) [reply src/TicketService.java:19]: Thank you so much! Well noted!

File: src/TicketService.java (hunk 2 above)
Comment (MadinaBaizhanova) [reply src/TicketService.java:48]: Thank you so much! Well noted!

**Pull request closed without merging**